triplea_maps.yaml

```diff
@@ -317,6 +317,22 @@
     <br>http://tripleadev.1671093.n2.nabble.com/World-War-II-Global-1940-Original-tp7314662p7314662.html
     <br>
   version: 3.9
+- mapName: World War II Global Balanced
+  url: https://github.com/regularkyd/G40Balanced/blob/master/World%20War%20II%20G40%20Balanced.zip
```

Comment:
Change the link to: https://github.com/triplea-maps/world_war_ii_g40_balanced/releases/download/0.3/world_war_ii_g40_balanced.zip